test/slow/api_spec_slow.js

```diff
@@ -197,15 +197,43 @@ describe(`electron-forge API (with installer=${installer.substr(12)})`, () => {
       const genericTargets = fs.readdirSync(path.resolve(__dirname, '../../src/makers/generic')).map(file => path.parse(file).name);
 
       [].concat(targets).concat(genericTargets).forEach((target) => {
-        describe(`make (with target=${target})`, () => {
-          before(async () => {
-            const packageJSON = await readPackageJSON(dir);
-            packageJSON.config.forge.make_targets[process.platform] = [target];
-            await fs.writeFile(path.resolve(dir, 'package.json'), JSON.stringify(packageJSON));
-          });
-
-          it('successfully makes with default config', async () => {
-            await forge.make({ dir, skipPackage: true });
+        [{
+          format: 'missing',
+          value: undefined,
+          expectFailure: true,
+        }, {
+          format: 'simple-string',
+          value: 'First Last',
+        }, {
+          format: 'full-string',
+          value: 'First Last <first.last@example.com> (http://www.example.com/)',
+        }, {
+          format: 'object',
+          value: {
+            name: 'First Last',
+            email: 'first.last@example.com',
+            url: 'http://www.example.com/',
+          },
+        }].forEach((author) => {
```

Comment:
I 100% agree ([see above comment](https://github.com/electron-userland/electron-forge/pull/161#issuecomment-284772793)), but at first glance I didn't see a way to do this without changing the API. On second thought, if the test could inject a mock `electron-windows-store` then it could just check the `opts` parameters it received without actually performing a build. 

If anyone would be willing to help me learn to do this please contact me on Slack (electron or ember teams) and I'll improve these tests.